semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbTrees.java

```diff
@@ -160,6 +161,9 @@ private Semanticdb.Tree annotationParameter(ExpressionTree expr) {
           unaryOpTree(
               semanticdbUnaryOperator(unaryExpr.getKind()),
               annotationParameter(unaryExpr.getExpression())));
+    } else if (expr instanceof ParenthesizedTree) {
+      ParenthesizedTree parenExpr = (ParenthesizedTree) expr;
+      return annotationParameter(parenExpr.getExpression());
```

Comment:
@olafurpg can you confirm it's fine to drill into subtree without creating intermediate semanticdb nodes?

Reply:
This is fine. No need to encode parentheses here, we can add it later if necessary